src/arrayMethodSort.js

```diff
@@ -4,8 +4,31 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction) {
+    const defaultCompareFun = (a, b) => {
+      const strA = String(a);
+      const strB = String(b);
+
+      return strA < strB ? -1 : strA > strB ? 1 : 0;
```

Comment:
The defaultCompareFun function works, but keep in mind that converting numbers to strings for comparison can lead to unexpected results when sorting numbers. For example, '10' will come before '2' when sorted as strings. It's not an issue if the function is used with the intention to sort strings.